commitlint.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
```

Comment:
Does it mean that every commit will have to follow this standard?
That will be annoying for individual PRs
Can we have this only for PRs and/or merges to develop & main?